main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
```

Comment:
Consistency is key. You have a discrepancy on when you use a space and when you don't. This works, but readability is improved when you choose a consistent stylization.
```suggestion
    print("Be kind to your", noun, "- footed", plural_noun)
```